packages/components/src/playground/button.spec.tsx

```diff
@@ -1,38 +1,43 @@
+import { primary, secondary } from "./button.stories";
 import { axe } from "jest-axe";
+import { prepareStory } from "@chanzuckerberg/story-utils";
 import { render } from "@testing-library/react";
-import { variants } from "./button.stories";
 
 describe("<Button />", () => {
-  it("renders the component", () => {
-    const { container } = render(variants());
+  it("renders primary buttons", () => {
+    const { container } = render(prepareStory(primary));
```

Comment:
probably because it's not published yet, but this could be reduced down to the `generateSnapshots`

Reply:
oh it's ready now! I can update it for lp-design-system

Reply:
Cool 👍 . I'm going to leave that for a follow up, because I'm lazy 😸